master/getting-started/docker/index.md

```diff
@@ -1,4 +1,29 @@
 ---
 title: Calico with Docker
 ---
-More information coming soon!
+
+Calico drives container connectivity and implements network policy for Docker containers.
+
+Use the navigation bar on the left to view information on Calico for Docker,
+or continue reading for an overview of recommended guides to get started.
+
+
+## Installation
+
+#### [Prerequisites](installation/prerequisites)
+
+Information on running etcd and configuring Docker for multi-host networking.
+
+#### [Manual Install  Guide]({{site.baseurl}}/{{page.version}}/getting-started/docker/installation/manual)
+
+Learn the two-step process for launching Calico for Docker.
```

Comment:
This method walks through the necessary manual steps to integrate Calico with Docker networking. Follow this guide if you’re integrating Calico with your own Docker orchestration system.